sources/packages/web/src/components/layouts/Institution/sidebar/ManageInstitutionSideBar.vue

```diff
@@ -11,6 +11,15 @@
         :title="item.label"
       />
     </v-list>
+    <template #append>
+      <v-list density="compact" nav>
+        <v-list-item
+          :to="{ name: InstitutionRoutesConst.OFFERINGS_UPLOAD }"
+          title="Offerings upload"
+          prepend-icon="fa:far fa-address-card-o"
+        />
```

Comment:
fyi- i have added a style to reduce space between icon and label and to get rid of the trimming(non-wrap) of label. you can use ` class="no-wrap"` for `v-tem`